extensions/passwords.py

```diff
@@ -0,0 +1,38 @@
+# Nextcord
+from nextcord.ext import commands
+
+# Randoms
+import random
+
+# Variabless
+lower = 'abcdefghijklmnopqrstuvwxyz'
+upper = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+numbers = '0123456789'
+symbols = '!@#$%^&'
+
+
+# commands
+class Passwords(commands.Cog):
+    def __init__(self, bot):
+        self.bot: commands.Bot = bot
+
+    @commands.command(
+        aliases=['pass', 'word']
+    )
+    async def password(self, ctx: commands.Context):
+        all = lower + upper + numbers + symbols
+        length = 9
```

Comment:
Maybe making the length a parameter?

Reply:
So you want the length to be modifiable?

Reply:
I didn't make the length modifiable, because if they wanted to make it longer, shorter or anything else, they could go to the page you made.

Reply:
That's kinda like a promotion, I mean, I don't think you want to go to a website for a password. Make the length modificable, but not freely. Try to make it with a minimum of 8 and maximum of 16

Reply:
Yes, you're right, I'll try